### PR TITLE
Created tablet and desktop versions of portfolio

### PR DESCRIPTION
**Pull request by Daf98**
- Added media query for desktop screens
- Added media query for tablet screens
- Fixed responsiveness issues

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,13 +43,15 @@ <h3 class="subtitle">I'm a Software Developer</h3>
     <li><a href="https://medium.com/"><img src="./icons/Medium.png" alt="Medium icon"></a></li>
 </ul>
 </section>
```
[the remark this answers is not in the document]
Comment (Daf98) [reply index.html:45]: what size screen are you viewing this on?

File: index.html [hunk 2]
```diff
@@ -14,11 +14,20 @@
...
             <img src="icons/Union.png" alt="Hamburger menu icon">
         </nav>
+        <nav id="container">
+            <ul class="displayedMenu"> 
+                <li><a href=""></a>Portfolio</li>
+                <li><a href=""></a>About</li>
+                <li><a href=""></a>Contact</li>
+                <li><img src="./icons/Disabled.png" alt="Envelope"></li>
+            </ul>
+        </nav>
```
[the remark this answers is not in the document]
Comment (Daf98) [reply index.html:28]: @chaw-bot There is no x-overflow on my screen, no matter how small or big I make it 
image

File: index.html [hunk 3]
```diff
@@ -65,94 +80,125 @@ <h2 class="bigText">Profesional Art Printing Data</h2>
...
             <li class="boxes">HTML</li>
             <li class="boxes">Bootstrap</li>
             <li class="boxes">Ruby</li>
         </ul>
+        <button type="button" class="regButton">See Project</button>
     </div>
-    <button type="button" class="regButton">See Project</button>    
+        
 </div>
+</section>
```
[the remark this answers is not in the document]
Comment (Daf98) [reply index.html:142]: @chaw-bot I cannot change the button on the card without JavaScript.

**Review by zhadier — APPROVED** [review]
## Hi @Daf98 👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 



### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

### P.S Remember to add the Get Started Section before you merge!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**